app/jobs/geocode_job.rb

```diff
@@ -4,7 +4,7 @@ class GeocodeJob < ApplicationJob
   def perform(klass, id)
     record = klass.classify.safe_constantize.find(id)
     results = Geocoder.search(record.full_address)
-    if results
+    if results && !results.empty?
```

Comment:
```suggestion
    if results.present?
```

Wouldn't that be the same thing?

Reply:
👍